### PR TITLE
[GEOS-10440] Documentation updates to reflect changes to the Elasticsearch module …

### PR DESCRIPTION
**Pull request by turingtestfail**
[GEOS-10440](https://osgeo-org.atlassian.net/browse/GEOS-10440)

…in GeoTools

removed editor file

  

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/main/CONTRIBUTING.md).
- [x] I have sent a [Contribution Licence Agreement](https://docs.geoserver.org/latest/en/developer/policies/committing.html) (not required for small changes, e.g., fixing typos in documentation).
- [x] First PR targets the `main` branch (backports managed later; ignore for branch specific issues).
- [x] All the build checks are green ([see automated QA checks](https://docs.geoserver.org/latest/en/developer/qa-guide/index.html)).

For core and extension modules:

- [ ] New unit tests have been added covering the changes.
- [x] [Documentation](https://github.com/geoserver/geoserver/tree/main/doc/en/user/source) has been updated (if change is visible to end users).
- [ ] The [REST API docs](https://github.com/geoserver/geoserver/tree/main/doc/en/api/1.0.0) have been updated (when changing configuration objects or the REST controllers).
- [x] There is an issue in the [GeoServer Jira](https://osgeo-org.atlassian.net/browse/GEOS/summary) (except for changes that do not affect administrators or end users in any way).
- [x] Commit message(s) must be in the form ``[GEOS-XYZWV] Title of the Jira ticket``.
- [x] Bug fixes and small new features are presented as a single commit.
- [x] Each commit has a single objective (if there are multiple commits, each has a separate JIRA ticket describing its goal).

### CODE REVIEW

File: doc/en/user/source/community/elasticsearch/index.rst [hunk 1]
```diff
@@ -223,29 +223,80 @@ BBOX and CQL filter::
...
+     </NamedLayer>
+    </StyledLayerDescriptor>
 
-Note that commas in native queries must be escaped with a backslash.
 
 Aggregations
 ------------
 
-Elasticsearch aggregations are supported through WFS/WMS requests by including the ``a:{aggregation_body}`` key:value pair in the ``viewparams`` parameter (see GeoServer SQL Views documentation for more information)::
+Elasticsearch aggregations are supported through WMS requests by by including the query in WMS requests through a custom rendering transformation, ``vec:GeoHashGrid``, which translates aggregation response data into a raster for display.
```
Comment (aaime) [inline doc/en/user/source/community/elasticsearch/index.rst:297]: "requests by by including".

Bye bye! :-D

**Review by aaime — CHANGES_REQUESTED** [review]

File: doc/en/user/source/community/elasticsearch/index.rst [hunk 2]
```diff
@@ -223,29 +223,80 @@ BBOX and CQL filter::
...
+       <UserStyle>
+         <Title>Test</Title>
+         <Abstract>Test Native Query</Abstract>
+         <FeatureTypeStyle>
+           <Transformation>
+             <ogc:Function name="vec:GeoHashGrid">
+               <ogc:Function name="parameter">
+                 <ogc:Literal>data</ogc:Literal>
+               </ogc:Function>
+			         <ogc:Function name="parameter">
```
Comment (aaime) [inline doc/en/user/source/community/elasticsearch/index.rst:247]: Formatting is not aligned here,

File: doc/en/user/source/community/elasticsearch/index.rst (hunk 1 above)
Comment (turingtestfail) [reply doc/en/user/source/community/elasticsearch/index.rst:297]: Fixed

File: doc/en/user/source/community/elasticsearch/index.rst (hunk 2 above)
Comment (turingtestfail) [reply doc/en/user/source/community/elasticsearch/index.rst]: Fixed

**Review by aaime — APPROVED** [review]

**Pull request merged**